App/views/user_view.py

```diff
@@ -1,5 +1,5 @@
 from flask.blueprints import Blueprint
 from App.controllers.user import display_all_users
```

Comment:
will this import not fail?

Reply:
Strange, I removed this file in my fixes though.
It's marked as outdated and it's not in my local branch, I'll abandon the PR and re-create.